index.html

```diff
@@ -0,0 +1,263 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.css">
+    <link rel="icon" href="images/fav-icon.ico">
+    <link rel="stylesheet" href="style.css">
+    <title>Document</title>
+</head>
+<body>
+    <div class="ham-menu ham-gone">
+        <a href="#" id="ham-close"><img class="ham-menu-close" src="images/pop-close.svg" alt="close-icon"></a>
+        <ul>
+            <li><a class="ham-links" href="index.html"> Home</a></li>
+            <li><a class="ham-links" href="about.html"> About</a></li>
+            <li class="ham-links">Program</li>
+        </ul>
+    </div>
+    <section class="first-section">
+        <nav>
+            <a href="#" class="ham" id="ham"><img class="hamburger" src="images/hamburger-menu.png" alt="hambuger-icon"></a>
+            <div class="desk-nav">
+                <div class="fixed-top">
+                    <ul class="fixed-ul">
+                        <li><i class="fa fa-facebook" aria-hidden="true"></i></li>
+                        <li><i class="fa fa-twitter" aria-hidden="true"></i></li>
+                        <li>English</li>
+                        <li>My Page</li>
+                        <li>Logout</li>
+                    </ul>
+                </div>
+                <div class="main-nav">
+                    <div class="desk-nav-bg">
+                        <div class="logo-div">
+                            <a href="index.html">
+                                <img class="logo-image" src="images/my-logo-1-x.png" alt="Summit-logo">
+                            </a>
+                            
+                        </div>
+                        <div class="menu-items">
+                            <ul>
+                                <li><a class="about-link" href="about.html"> About</a></li>
+                                <li>Program</li>
+                                <li>Join</li>
+                                <li>Sponsor</li>
+                                <li>News</li>
+                            </ul>
+                            <button class="menu-button">AE Summit</button>
+                        </div>
+                    </div>
+                </div>
+            </div>
+        </nav>
+       <div class="intro-section">
+            <div class="intro-texts">
+                <h3 class="text-excl main-color">"Hello! Agriculturist"</h3>
+                <h1 class="top-header">AGRICULTURAL ENTHUSIASTS GLOBAL SUMMIT 2023</h1>
```

Comment:
- [ ] The first `<hx>` is a text which means it should be in the `<p>` tag instead of the `<hx>` tag, kindly fix that in the about.html file as well 🙃 
- [ ] Also I would like to suggest to reduce a little bit the size of the two lines, the text and the page title. They are too big, for the `"Hello! Agriculturist"` you can reduce to `2.5em` instead of `3em` and for the title `3.2em` instead of `5em` 🙃 🙃 

Before

<img width="914" alt="Screenshot 2023-04-20 at 20 13 40" src="https://user-images.githubusercontent.com/74501523/233439864-5fc41d00-73cc-4beb-9efe-46ea7978bc77.png">

After

<img width="914" alt="Screenshot 2023-04-20 at 20 13 24" src="https://user-images.githubusercontent.com/74501523/233440051-4b545e4f-40cc-4942-be90-a146244c2c06.png">